### PR TITLE
Bloat removal v2

### PR DESCRIPTION
**Pull request by CrytoPal**
## Type of change

- [ ] Bug fix
- [ ] New feature

## Description

Removes Offhand Crash and Slippery

## Related issues

nothing

# How Has This Been Tested?

Multiplayer

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have added comments to my code in more complex areas.
- [ ] I have tested the code in both development and production environments.

### CODE REVIEW

**Comment by Fiz-Victor** [comment]
Cant you remove them in one commit?

**Comment by CrytoPal** [comment]
I reset my pc, and I just recently got git back

**Comment by ridglef** [comment]
fuck you i love slippy

**Comment by Fiz-Victor** [comment]
anyway offhand crash is broken on paper servers (could be removed)

File: src/main/java/meteordevelopment/meteorclient/mixin/BlockMixin.java [hunk 1]
```diff
@@ -37,20 +36,4 @@ private static boolean onShouldDrawSide(boolean original, BlockState state, Bloc
...
-        if (Modules.get() == null) return original;
-
-        Slippy slippy = Modules.get().get(Slippy.class);
-        Block block = (Block) (Object) this;
-
-        if (slippy.isActive() && (slippy.listMode.get() == Slippy.ListMode.Whitelist ? slippy.allowedBlocks.get().contains(block) : !slippy.ignoredBlocks.get().contains(block))) {
-            return slippy.friction.get().floatValue();
-        }
-
-        if (block == Blocks.SLIME_BLOCK && Modules.get().get(NoSlow.class).slimeBlock()) return 0.6F;
```
Comment (Fiz-Victor) [inline src/main/java/meteordevelopment/meteorclient/mixin/BlockMixin.java:53]: you've just removed the NoSlow code bro

**Pull request closed without merging**